.github/workflows/maven.yml

```diff
@@ -27,5 +25,9 @@ jobs:
         java-version: '21'
         distribution: 'zulu'
         cache: maven
-    - name: Build with Maven
-      run: mvn clean install
+    - name: Build
```

Comment:
Please do a rebase with the main branch so these changes won't show up on the PR anymore